data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2471 @@
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper10 static"
+		"submunition" "creeper11 print"
+		
+outfit "creeper10 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+
+outfit "creeper11 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper11 static"
+		"submunition" "creeper12 print"
+		
+outfit "creeper11 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 25
+		"submunition" "asteroid missile"
+
+outfit "creeper12 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper12 static"
+		"submunition" "creeper13 print"
+		
+outfit "creeper12 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 20
+		"submunition" "asteroid missile"
+
+outfit "creeper13 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper13 static"
+		"submunition" "creeper14 print"
+		
+outfit "creeper13 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 15
+		"submunition" "asteroid missile"
+
+outfit "creeper14 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper14 static"
+		"submunition" "creeper15 print"
+		
+outfit "creeper14 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 10
+		"submunition" "asteroid missile"
+
+outfit "creeper15 print"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"velocity" 2
+		"acceleration" 3
+		"drag" .1
+		"turn" -8
+		"lifetime" 5
+		"submunition" "creeper15 static"
+		
+outfit "creeper15 static"
+	weapon
+		sprite "effect/asteroid projectile static"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" .0000001
+		"drag" 1
+		"lifetime" 5
+		"submunition" "asteroid missile"
+
+outfit "asteroid missile"
+	weapon
+		sprite "effect/asteroid projectile print"
+			"start frame" 1
+			"frame rate" 9
+		"acceleration" 2
+		"drag" .1
+		"turn" 6
+		"homing" 3
+		"tracking" 1
+		"lifetime" 120
+
+outfit "Asteroid Weapon Large 2"
```

Comment:
If I haven't made a mistake in reading this (which is unlikely), this fires off a total of 31 `"asteroid laser 1"`s per shot. 31x1.7 = 52.7 discharge damage and 182.9 hull damage. The hull damage is very modest, the discharge damage less so (as it's effectively 5270 shield damage). At 0.4 shots/sec, this is a little over 1000 shield DPS - very strong, but certainly not unbalanceable.